github2fedmsg/views/webhooks.py

```diff
@@ -132,11 +136,19 @@ def webhook(request):
     fas_usernames = build_fas_lookup(payload)
     payload['fas_usernames'] = fas_usernames
 
-    fedmsg.publish(
-        modname="github",
-        topic=event_type,
-        msg=payload,
-    )
+    try:
+        msg = Message(
+            topic="github.{}.v1".format(event_type),
```

Comment:
Is this the version of the schema or does this mean something else?

Reply:
Yes, except we have decided that for now the topics should not change, so I should remove it. Later, when everybody's using fedora-messaging, we'll be able to add versions to the topic to signify a change is schema.

Reply:
Shouldn't this be taken directly from schema package? Like version in header field? Because you can easily forgot to change it, if it will be like this in the code.

Reply:
This could be a way to do it. We could automatically append the version to the topic before sending. The thing is that fedmsg did a lot of topic wrangling behind the scenes and it made things very complicated when you wanted to subscribe to a specific topic. If we add the version automatically, subscribers will have to remember to add the topic version to their configuration file, and they may forget that too. There are pros and cons on each side, we have opted for expliciteness (for now).